lib/addons/xray/xray-ds.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
```

Comment:
where did this come from? Is there a Helm chart we can use?

Reply:
I used the one from the EKS Workshop: https://www.eksworkshop.com/intermediate/245_x-ray/x-ray-daemon/ 
Unfortunately, there is no helm chart. The approach in the official documents is even more horrible than the one used in the workshop since it is not using IRSA.